instrumentation-genai/opentelemetry-instrumentation-vertexai/tests/test_instrumentor.py

```diff
@@ -0,0 +1,37 @@
+import pytest
+from google.cloud.aiplatform_v1.services.prediction_service import client
+from google.cloud.aiplatform_v1beta1.services.prediction_service import (
+    client as client_v1beta1,
+)
+
+from opentelemetry.instrumentation.vertexai import VertexAIInstrumentor
+
+
+@pytest.fixture(name="instrumentor")
+def fixture_instrumentor():
```

Comment:
Shouldn't this be the fixture you use in all the tests?

Reply:
Re-used from conftest.py thanks.